src/langchain_google_cloud_sql_pg/engine.py

```diff
@@ -544,11 +550,13 @@ def init_vectorstore_table(
                 metadata. Default: []. Optional.
             metadata_json_column (str): The column to store extra metadata in JSON format.
                 Default: "langchain_metadata". Optional.
-            id_column (str):  Name of the column to store ids.
-                Default: "langchain_id". Optional,
+            id_column (Union[str, Column]) : Column to store ids.
+                Default: "langchain_id" column name with data type UUID. Optional.
             overwrite_existing (bool): Whether to drop existing table. Default: False.
             store_metadata (bool): Whether to store metadata in the table.
                 Default: True.
+        Raises:
+            :class:`UndefinedObjectError <asyncpg.exceptions.UndefinedObjectError>`: if id column data type is invalid.
```

Comment:
nit: "invalid" is vague and does not help user understand what this error suggest. What is deemed invalid?
```suggestion
           UndefinedObjectError: if id column data type is invalid.
```